bcgov/registry-schemas
# MHR API qualified supplier updates

*Issue #:* /bcgov/entity#17306

*Description of changes:*
Latest MHR API schema updates:
- registrationSummary schema add frozenDocumentType
- registration schema  add frozenDocumentType
- Create qualifiedSupplier schema (new dbaName not in common party schema)
- Create termsSummary schema
- Update manufacturerInfo schema add termsAccepted, dbaName


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the registry-schemas license (Apache 2.0).
